docs/source/index.rst

```diff
@@ -75,6 +77,12 @@ or install it using your system package manager, e.g.::
 
     sudo apt-get install python3-pyqt5
 
+or::
+
+    sudo apt-get install python-pyqt
+    sudo apt-get install python3-pyqt
+
+if PyQt4 is desired.
```

Comment:
The reason I'd rather not put them in is that once we have more than one example, there's no obvious reason why we're not adding others. If PyQt4 is worth specific mention, why not PySide? Why not commands for Fedora, or Arch Linux?

I also think that when we write docs, we have a tendency to leave in extra information because of a vague feeling that 'more information never hurts', and that gradually leads to longwinded docs that no-one can be bothered to read. So I try to resist that.